airbyte-config/config-models/src/main/java/io/airbyte/config/EnvConfigs.java

```diff
@@ -1043,11 +1042,6 @@ public Set<Integer> getTemporalWorkerPorts() {
     return Arrays.stream(ports.split(",")).map(Integer::valueOf).collect(Collectors.toSet());
   }
 
-  @Override
```

Comment:
I think that is is missing an update to the `Configs.java` file as well

Reply:
👍